packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -632,7 +632,6 @@ function getRank(
  * public-instance-method.
  * If a lowest possible rank is a member group, a comma separated list of ranks is returned.
  * @param ranks the existing ranks in the object.
- * @param target the target rank.
```

Comment:
This doesn't look very useless to me? And to me omitting docs for one parameter looks like a greater evil...